### PR TITLE
UpExtreme: add support for mockup device

### PR DESCRIPTION
**Pull request by plbossart**
Add pretend devices to probe dummy SoundWire devices and load a
machine driver.

FIXME: the device _ADR has to be fixed.

Signed-off-by: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>

### CODE REVIEW

File: UpExtreme/mockup-device-1amp.asl [hunk 1]
```diff
@@ -0,0 +1,40 @@
+ /** @file
+  The definition block in ACPI table for RT5682 under SoundWire Controller
+  (link2)
```
Comment (bardliao) [inline UpExtreme/mockup-device-1amp.asl]: Maybe edit it to a suitable description.

File: UpExtreme/mockup-device-1amp.asl [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  Copyright (c) 2019, Intel Corporation. All rights reserved.<BR>
```
Comment (bardliao) [inline UpExtreme/mockup-device-1amp.asl]: 2021?

File: UpExtreme/mockup-device-1amp.asl [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+**/
+
+DefinitionBlock (
+  "",
+  "SSDT",
+   2,
+  "INTEL",
+  "RTK0Tabl",
```
Comment (bardliao) [inline UpExtreme/mockup-device-1amp.asl]: INTTabl? I have no idea what is the suitable name.

**Review by bardliao — COMMENTED** [review]
@plbossart Overall it looks good to me, but should we make the description and the naming more suitable?

File: UpExtreme/mockup-device-1amp.asl (hunk 1 above)
Comment (plbossart) [reply UpExtreme/mockup-device-1amp.asl]: yes, fixed

File: UpExtreme/mockup-device-1amp.asl (hunk 2 above)
Comment (plbossart) [reply UpExtreme/mockup-device-1amp.asl]: yes!

File: UpExtreme/mockup-device-1amp.asl (hunk 3 above)
Comment (plbossart) [reply UpExtreme/mockup-device-1amp.asl]: I chose CML1 for 1-amp

**Pull request merged**